src/core/stdc/config.d

```diff
@@ -120,6 +120,8 @@ else version( DigitalMars )
             alias real c_long_double;
         else version( OSX )
             alias real c_long_double;
+        else version( iOS )
```

Comment:
No real reason besides some of the one liners I got lazy on.  I suppose with watchOS and tvOS getting support later, probably should follow the same pattern of defining version Darwin and using that.